backend/core/types/src/backend-swagger.ts

```diff
@@ -3726,6 +3726,9 @@ export interface ListingFilterParams {
 
   /**  */
   maxRent?: number
+
+  /**  */
+  ami?: string
```

Comment:
number?

Reply:
oh it should be, this is from the other thing you noticed in listing.dto where it was the wrong type in the decorator.